SwinMM/WORD/README.md

```diff
@@ -0,0 +1,42 @@
+# Note for SwinMM Finetuning
+
+## Training
+
+### FIXME(outdated)
```

Comment:
I guess this would need some updates

Reply:
This file is out of date and I have removed it. All the training details can be find in the TRAINING.md